components/Academy/ApplicationProcess/ApplicationProcess.module.scss

```diff
@@ -0,0 +1,126 @@
+.container {
+    padding: $grid $grid * 1.5 $grid * 2;
+
+    @include media-query(medium) {
+        padding: $grid * 2 $grid * 3;
+    }
+
+    @include media-query(large) {
+        padding: 0 $grid * 3 $grid * 4 $grid * 3;
+    }
+
+    @include media-query(menu-break) {
+        padding: 0 $grid * 6 $grid * 4 $grid * 4;
+    }
+}
+
+.introductionContainer {
+    display: flex;
+    flex-direction: column;
+    @include media-query(large) {
```

Comment:
**Nickpick**: We want a blankline above this for consistency, I can't seem to find the right eslint rule for this however.